inst/tests/tests.Rraw

```diff
@@ -13947,13 +13947,15 @@ test(1981.1, DT[, new_col := shift(val, "lead")],      error="is.numeric(n) is n
 test(1981.2, DT[, new_col := shift(val, NA_integer_)], error="Item 1 of n is NA")
 
 # print of DT with many columns reordered them, #3306.
+ptc = options(datatable.print.trunc.cols=FALSE) #4552
```

Comment:
shouldn't we rather put this in R/test.data.table.R along with all the other options set for `test.data.table()`?

Reply:
Good spot. Done.